### PR TITLE
Updated CONTRIBUTION.md

### LINKED ISSUE #1225: Add packages for ARM based devices
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [CONTRIBUTING.md](https://github.com/publiclab/image-sequencer/blob/main/CONTRIBUTING.md) by adding the packages `xserver-xorg-dev libxext-dev1` and the `--fix-missing` flag to the command `sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config`.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by deveshkau**
Fixes #1225  

I have update CONTRIBUTION.md with `xserver-xorg-dev libxext-dev` packages.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Happily accepting pull requests; to edit the core library, modify files in `./sr
 
 On ARM based devices, the `gl` module may require some libraries to be re-installed:
 
-`sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
+`sudo apt-get install --fix-missing -y build-essential xserver-xorg-dev libxext-dev libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
```
Comment (HarshKhandeparkar) [inline CONTRIBUTING.md]:
```suggestion
`sudo apt-get install -y build-essential xserver-xorg-dev libxext-dev libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
```
I don't think the fix-missing is required.

Comment (deveshkau) [reply CONTRIBUTING.md]: @HarshKhandeparkar thank you for helping. I got a pretty good idea of raising pull-request and working of Travis-ci. It was an awesome experience. Looking forward to more contribution in the future.

**Comment by deveshkau** [comment]
@publiclab/is-reviewers

**Comment by deveshkau** [comment]
@jywarren @rexagod @publiclab/is-reviewers Please review.

**Comment by jywarren** [comment]
Thank you!!!!

**Pull request merged**